docs/index.md

```diff
@@ -1,38 +1,21 @@
+
```

Comment:
This additional blank line is causing a problem for Jekyll: the .html file isn't generated. If I remove the blank line, I can generate the site.

BTW, I'm running the site like this:

```
rm Gemfile
docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll -it -p 4000:4000 jekyll/jekyll:pages jekyll s --watch --base
url '' --force_polling
```